SpaceCommander.podspec

```diff
@@ -14,5 +14,8 @@ Pod::Spec.new do |s|
   # Use this file to overcome CocoaPods limitation that at least one source file must be specified.
   s.source_files = "CocoaPodsDummy.m"
 
-  s.preserve_paths = "**/*"
+  s.preserve_paths = "**/*", ".clang-format"
+  
+  # Run setup-repo.sh when the pod is installed.
+  s.prepare_command = "./setup-repo.sh"
```

Comment:
Also, if this works, we should update the README to note that the install will happen automatically if Cocoapods are used

And we should add cocoapod installation instructions there too (the things described here can be a separate PR)